Userland/Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -1235,6 +1245,8 @@ static ErrorOr<void> process_chunk(Streamer& streamer, PNGLoadingContext& contex
         return process_fcTL(chunk_data, context);
     if (chunk_type == "fdAT"sv)
         return process_fdAT(chunk_data, context);
+    if (chunk_type == "eXIf"sv)
+        return process_eXif(chunk_data, context);
```

Comment:
Not sure if this matters but the case is different between eXIf and eXif.

Reply:
> Not sure if this matters
In practice I guess no, but they "eXif" and "eXIf" are technically different chunks.
Thanks for catching that!